src/viewer.js

```diff
@@ -1217,6 +1218,10 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
 
         this._hideMessage();
 
+        $.extend ( true, options, {
```

Comment:
Oh damn... Thanks a lot, i had not tried this functionality yet... I was very confused when passing the options :laughing: